client/elements/addons/sc-top-sheet-views.js

```diff
@@ -330,6 +338,31 @@ export class SCTopSheetViews extends LitLocalized(LitElement) {
     super.connectedCallback();
     this._loadScToasts();
     this._fetchReferenceDisplayType();
+    document.addEventListener('keydown', this._handleKeydown.bind(this));
+    window.setTimeout(() => {
+      this.show();
+      this.hide();
+    }, 2000);
+  }
+
+  // This componenet is _never_ disconnected the way the code is written today (2024-08-04)
+  // Still, adding this in case some day this component is properly unloaded.
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    document.removeEventListener('keydown', this._handleKeydown);
+  }
+
+  _handleKeydown(event) {
+    switch (event.key) {
+      case 'v':
+      case 'V':
+        var idx = this.textViewArray.findIndex(item => item.textView === this.selectedTextView);
```

Comment:
Using `const` or `let` instead of `var` is more in line with the specification 🙏🏼